libs/sdk-core/src/breez_services.rs

```diff
@@ -1368,6 +1368,38 @@ impl BreezServices {
             },
         })
     }
+
+    pub async fn create_offer(&self, req: CreateOfferRequest) -> Result<String> {
```

Comment:
This API is actually similar to `receive_payment` (that returns bolt11) but for bolt12.
The way it is right now is a bit inconsistent since the `receive_payment` returns a response with the invoice parsed (LNInvoice) and here we return just the raw offer. For consistency perhaps we should return the LNOffer structure here.